app/assets/stylesheets/style.scss

```diff
@@ -9987,8 +9987,8 @@ body {
 #mainNav .navbar-toggler {
   font-size: 80%;
   padding: 0.75rem;
-  color: #64a19d;
-  border: 1px solid #64a19d;
+  color: #ee9d1c;
```

Comment:
Color now matches the yellow/mustard on the page

Reply:
better than bisque